mypy/types.py

```diff
@@ -542,6 +543,13 @@ def deserialize(cls, data: JsonDict) -> 'FunctionLike':
 _dummy = object()  # type: Any
 
 
+FormalArgument = NamedTuple('FormalArgument', [
+    ('name', str),
```

Comment:
Use `Optional[...]` for types of items that can be `None` (for documentation, these aren't checked yet).